content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -1,95 +1,93 @@
-=========================
-Create Bills of Materials
-=========================
+==========================
+Create a Bill of Materials
+==========================
 
-A *Bill of Materials* is a document defining the quantity of each
-component required to make or deliver a finished product. Additionally, it 
-can also include various operations and individual step guidelines
-needed to complete a production process. 
+A *Bill of Materials* (:abbr:`BoM (Bill of Materials)`) is a document that defines the quantity of
+each component required to make or deliver a finished product. It can also include various
+operations and the individual step guidelines needed to complete a production process.
 
-With Odoo's MRP solution, multiple BOMs can also be linked to each product,
-so that even product variants can have their *own* tailored BOM.
+In Odoo Manufacturing, multiple :abbr:`BoMs (Bills of Materials)` can be linked to each product, so
+that even product variants can have their own tailored :abbr:`BoMs (Bills of Materials)`.
 
-This will help you to optimize your manufacturing process and save time 
-as a result. 
+Correctly setting up a :abbr:`BoM (Bill of Materials)` helps optimize the manufacturing process and
+save time.
 
-Setting up a BoM
-================
-The simplest BoM set up is one without operations or instructions. 
-In this case, you will manage your production using Manufacturing Orders
-only.
+Set up a Bill of Materials (BoM)
+================================
 
-To create a BoM from the Manufacturing module, go to :menuselection:`Products --> Bills of Materials`. 
-If you click **Create**, the first thing you'll have to do is to specify the final product. 
-For an existing product, simply select it from the list, or create a new one on the spot. Note 
-that if you go through the product form to create your BOM, the product will already be set for you.
+The simplest :abbr:`BoM (Bill of Materials)` setup is one without operations or instructions, only
+components. In this case, the production is solely managed using *Manufacturing Orders*.
 
-For a standard Bill of Material, keep the default *BoM* type, which is *Manufacture this Product*. 
-Now specify the various components that make up the production of your final product and their 
-respective quantities. You can create components as you go, on the BOM, or create products beforehand,
-by going to the :menuselection:`Top Menu --> Products --> Create`, and add them later on. 
+To create a :abbr:`BoM (Bill of Materials)` from the :guilabel:`Manufacturing` module, go to
+:menuselection:`Products --> Bills of Materials`. Then, click :guilabel:`Create`. Next, specify the
+:guilabel:`Product`. Note that a :abbr:`BoM (Bill of Materials)` can also be created directly from
+the product form. In that case, the :guilabel:`Product` field is pre-filled.
```

Comment:
Optional suggestions here:
- turn the sentence starting with `Note that a...` into an actual `:note:` since it's an alternate instruction (and therefore not needed in the paragraph with primary instruction / might confuse people).
- form the two short sentences into a compound sentence to retain context when vague nouns (e.g. "that" --> "which").

```suggestion
:guilabel:`Product`. 

.. :note::
   A :abbr:`BoM (Bill of Materials)` can also be created directly from the product form, in which
   case, the :guilabel:`Product` field is pre-filled.
```